### ISSUE TITLE
Dot indicators missing from welcome carousel, text needs vertical alignment

### ISSUE THREAD

**Issue by crieggalder** [andreisarebi/civic-app#32]
Current welcome carousel lacks dots indicating current screen of three screens. Text should be moved down to accommodate dot indicators.

Current:

![screenshot_20180916-163301](https://user-images.githubusercontent.com/36721640/45603239-022ab900-b9df-11e8-9f63-909759f6c4e0.png)

Dots from mocks:

![image](https://user-images.githubusercontent.com/36721640/45603248-1b336a00-b9df-11e8-9a0c-e4e0d83560da.png)

**Comment by gmferland** [comment]
I found that UI to be very tricky, so if anyone wants to revisit this go ahead. But it's very hard to put those dots in a place that looks good on all screens. And it's kind of hard as a developer to see what the UI will look like on any device

**Comment by crieggalder** [comment]
Makes sense! Definitely lower priority - going through and prioritizing issues now. Any way we can vertically center text between circle and the Register/Sign In menu?
